ssh/tailssh/tailssh_test.go

```diff
@@ -956,6 +968,12 @@ func TestSSHAuthFlow(t *testing.T) {
 			if len(tc.wantBanners) > 0 {
 				t.Errorf("missing banners: %v", tc.wantBanners)
 			}
+			if tc.authErr && !publicKeyUsed.Load() {
+				t.Error("public key should have been attempted")
+			}
+			if (tc.authErr || tc.usesPassword) && !passwordUsed.Load() {
```

Comment:
Funnily enough, the old test never checked with the password callback was actually used. This now checks for that.